composer.json

```diff
@@ -1,55 +1,59 @@
 {
-    "name": "acquia/acquia-ra-composer",
-    "description": "Project json for Acquia Remote Administration Composer builds",
-    "type": "project",
-    "license": "MIT",
-    "repositories": {
-        "drupal": {
-            "type": "composer",
-            "url": "https://packages.drupal.org/8"
-        }
+  "name": "acquia/acquia-ra-composer",
+  "description": "Project json for Acquia Remote Administration Composer builds",
+  "type": "project",
+  "license": "Proprietary",
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://packages.drupal.org/8"
     },
-    "require": {
-        "composer/installers": "^1.0",
-        "cweagans/composer-patches": "^1.6.0",
-        "drupal-composer/drupal-scaffold": "^2.0.0",
-        "drupal/admin_toolbar": "^1.0",
-        "drupal/core": "^8.4",
-        "drush/drush": "^9.0"
-    },
-    "require-dev": {
-        "drupal/devel": "^1.0",
-        "drupal/stage_file_proxy": "^1.0"
-    },
-    "conflict": {
-        "drupal/core": "7.*"
-    },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "extra": {
-        "installer-paths": {
-            "docroot/core": ["type:drupal-core"],
-            "docroot/libraries/{$name}": ["type:drupal-library","type:npm-asset","type:bower-asset"],
-            "docroot/modules/contrib/{$name}": ["type:drupal-module"],
-            "docroot/profiles/contrib/{$name}": ["type:drupal-profile"],
-            "docroot/themes/contrib/{$name}": ["type:drupal-theme"]
-        }
-    },
-    "scripts": {
-        "post-install-cmd": [
-            "@composer drupal-scaffold"
-        ],
-        "post-update-cmd": [
-            "@composer drupal-scaffold"
-        ],
-        "drupal-scaffold": [
-            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
-        ]
-    },
-    "enable-patching": true,
-	"patches": {
-		"drupal/[module_name]": {
-			"Note regarding the nature of the patch being applied": "https://www.drupal.org/files/issues/[patch_name].patch"     
-		}
-	}
+    {
+      "type": "composer",
+      "url": "https://asset-packagist.org"
+    }
+  ],
+  "require": {
+    "composer/installers": "^1.0",
+    "cweagans/composer-patches": "^1.6.0",
+    "drupal-composer/drupal-scaffold": "^2.0.0",
+    "drupal/admin_toolbar": "^1.0",
+    "drupal/core": "^8.3",
+    "drush/drush": "^8.1"
```

Comment:
Is this downgrade to Drush 8 intentional because it's causing **error** on DevDesktop since Drush 8 uses **Drush.launcher** which is not supported on Windows. It's recommended to use Drush 9 which doesn't have Drush.launcher?